src/pivot-table/components/cells/DataCell.tsx

```diff
@@ -45,13 +44,12 @@ const MeasureCell = ({ columnIndex, rowIndex, style, data }: MeasureCellProps):
     );
   }
 
-  const { qText, qType } = cell;
-  const isNull = qType === NxDimCellType.NX_DIM_CELL_NULL;
-  const isTotalValueCell = !isNull && isTotalValue(columnIndex, rowIndex);
-  const text = isNull ? layoutService.getNullValueText() : qText;
-  const isNumeric = isNull ? !Number.isNaN(+text) : true;
+  const { qText } = cell.ref;
```

Comment:
detail, but don't see why `qText` is destructured when isNull is not. That would remove the changes on the rows below as well. i think the syntax is like this

```suggestion
  const { isNull, ref: { qText } } = cell;
```

Reply:
Yeah I will actually remove the destruction. Don't think it adds any value there as `cell.ref.qText` is rather short and issue to read